simplic-data/Simplic.Data.Sql/SqlRepositoryBase.cs

```diff
@@ -40,21 +40,35 @@ public SqlRepositoryBase(ISqlService sqlService, ISqlColumnService sqlColumnServ
         /// <returns>Instance of <see cref="TModel"/> if exists</returns>
         public TModel Get(TId id)
         {
+            return GetByColumn<TId>(PrimaryKeyColumn, id);
+        }
+
+        /// <summary>
+        /// Get a model instance from the database, using a custom column name
+        /// </summary>
+        /// <typeparam name="T">Id type</typeparam>
+        /// <param name="columnName">Column name</param>
+        /// <param name="id">Id value</param>
+        /// <returns>Model if exists</returns>
+        protected TModel GetByColumn<T>(string columnName, T id)
```

Comment:
this would not work. this custom column could have a type other than T

Reply:
That is way the method is protected. E.g. if you want to implement something like `GetByname()`